Mathlib/Order/GaloisConnection.lean

```diff
@@ -781,6 +785,10 @@ theorem u_l_leftInverse [PartialOrder α] [Preorder β] (gi : GaloisCoinsertion
   gi.u_l_eq
 #align galois_coinsertion.u_l_left_inverse GaloisCoinsertion.u_l_leftInverse
 
+theorem u_bot [PartialOrder α] [Preorder β] [OrderBot α] [OrderBot β] (gi : GaloisCoinsertion l u) :
+    u ⊥ = ⊥ :=
+  gi.dual.l_top
+
```

Comment:
nit: can you put this above `u_l_leftInverse` so that the two sections are in the same order?

bors d+